### ISSUE TITLE
A pity that those repos are not integrated @scilifelab main org

### ISSUE THREAD

**Issue by brainstorm** [SciLifeLab/software-development#1]
https://www.gov.uk/service-manual/operations/devops.html

When one group owns a specific area (say quality) it’s easy for other areas to assume that it’s no longer their concern.

This attitude is toxic, especially in areas such as:
- quality
- release management
- performance

High quality digital services need to be able to adapt quickly to user needs, and this can only happen with close collaboration between different groups.

**Comment by robinandeer** [comment]
I'm totally up for moving it up to @SciLifeLab.

I mainly just set it up to have some place (which wasn't [AM Systems](https://amsystem.se/en/)...) to point SWEDAC and people here at Clinical that are somewhat new to Python etc.

I have to read up on your link, but do you also suggest we broaden the topic beyond general style guide/best practices?

It would be awesome to have you guys' input since I'm kind of the only one here with somewhat extensive Python experience :smiley:

cc @guillermo-carrasco

**Comment by guillermo-carrasco** [comment]
Hej @robinandeer and @brainstorm !

You're right Roman; Clinical, Production, NGI, Stockholm and Uppsala should collaborate closer in software development if we want to achieve software quality and better designs for our infrastructure. Robin attended the workshop in Best Practices in Development and actually I don't think he needed it :-), I'm quite impressed of his work, so I don't doubt that he will follow good practices and ask around if there are question. Robin you're welcome to do so.

The problem that I see (and I'm not saying that I'm happy with that) are things like strict QA systems or accreditations that forces us to kind of separate facilities so that the work of one facility does not interfere in the accreditation of the other. At least that's the case for production...

Even that, I feel like the communication between groups is better than a few months ago: Production and clinical are planning a shared instrument infrastructure, where we and Clinical will be able to use each other's instruments, Stockholm and Uppsala are developing together the pipeline for NGI projects, etc. I'm quite happy with that :-)

Maybe we should try to make more clear these collaborations?

**Comment by brainstorm** [comment]
Yes! I would totally make it more explicit and global @scilifelab. For instance, joining Trello boards for the different groups so that people are able to communicate, share and collaborate with tasks more easily and efficiently. It is definitely an issue worth raising to your favourite PI's and managers.

I bet @marcelm, from WABI is also interested in this thread, I talked with him recently about all this.

#breakdownsilos #devops

**Comment by robinandeer** [comment]
I'm pushing for Trello adoption @Clinical-Genomics. Hopefully at least the IT/bioinformatics side will start using it for real "soonish". After we get some more of our workflow into that system it would be interesting to tied it together with other Trello boards.

I don't have much experience with **accreditation** but my initial feeling is that as long as the information/routines are documented "somewhere" and versioned they shouldn't have to be discretely separated from other units. We have _copied_ a lot of documents from Production which if they (SWEDAC) really cared would sort of be just the same as sharing documents right out, no?

I think @brainstorm raises some very interesting ideas. Still not quite sure how this site/repo plays in to it fully, though. Perhaps it can be used to explore how more collaboration would work.

I'll bring it up with some manager :man: over here.

**Comment by robinandeer** [comment]
Some things are definitely moving forward but we are still not there yet. When it comes to this repo **I motion to retire it at the organisation level**. 

I will in the meantime continue to jot down my own conclusions about best practises and tips at http://www.robinandeer.com/software-development/.

If people are interested in bringing something like it back, just fork my repo when the time comes :)
